src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -178,6 +180,9 @@ class ReportActionItemMessageEdit extends React.Component {
         }
         if (e.key === 'Enter' && !e.shiftKey) {
             e.preventDefault();
+            if (this.state.draft.length > CONST.MAX_COMMENT_LENGTH) {
```

Comment:
Should we move this inside `publishDraft` just to keep it a little clean ?https://github.com/Expensify/App/blob/1e1cdcbcd49b3cab7dcd83c25be61d141b736c8a/src/pages/home/report/ReportActionItemMessageEdit.js#L133-L137

cc: @luacmartins

Reply:
I agree. @Puneet-here let's move this logic to `publishDraft`